exasol_script_languages_developer_sandbox/runtime/ansible/roles/jupyter/templates/etc/update-motd.d/999-jupyter

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import pathlib
+import json
+
+motd_message = """{{heading_jupyter_update_password}}
+
+Run as user {{user_name}}:
+"{{jupyterlab.virtualenv}}/bin/jupyter server password"
```

Comment:
I think, you first need to activate the virtual env, before you can use jt

Reply:
But, I saw the test, so it seems to work

Reply:
`/bin/jupyter` contains a shebang with the python interpreter of the virtualenv. Because of that, there is no need to activate the virtual-env.

Reply:
ah good to know